content/vault/v1.20.x/content/docs/platform/mssql/rotation.mdx

```diff
@@ -61,3 +61,16 @@ go
 
 ALTER DATABASE ENCRYPTION KEY ENCRYPTION BY SERVER ASYMMETRIC KEY TransitVaultAsymmetricV2;
 ```
+
+The above steps will wrap the DEK with the latest version of the specified Transit key as the asymmetric KEK.  To use a different version of the Transit key, please specify the version as shown below.
+
+```sql
+
+CREATE ASYMMETRIC KEY TransitVaultAsymmetricV1
+FROM PROVIDER TransitVaultProvider
+WITH CREATION_DISPOSITION = OPEN_EXISTING,
+PROVIDER_KEY_NAME = 'ekm-encryption-key?version=1';
+```
```

Comment:
Delete the extra line

```suggestion
```sql
CREATE ASYMMETRIC KEY TransitVaultAsymmetricV1
FROM PROVIDER TransitVaultProvider
WITH CREATION_DISPOSITION = OPEN_EXISTING,
PROVIDER_KEY_NAME = 'ekm-encryption-key?version=1';
```
```